wiki/Music_theory/Beat/id.md

```diff
@@ -0,0 +1,5 @@
+# Ketukan (Beat)
+
+Sebuah **ketukan (beat)** adalah satuan waktu yang berulang dalam musik, khususnya denyut nadi musik yang diketuk oleh pendengar. Di osu!, kamu dapat membandingkannya dengan tanda tick putih di [editor timeline](/wiki/Beatmap_Editor/Timelines) yang menjadi kekuatan irama lagu.
+
+![Sebuah screenshot dalam permainan pada editor timeline](img/timeline.png "Semua tanda tick putih mewakili irama dalam musik")
```

Comment:
"Screenshot" should be translated as well for consistency i suppose

`Sebuah screenshot dalam permainan pada editor timeline` --> i.e. `Cuplikan kolom timeline pada editor`